server/src/test/java/org/candlepin/controller/PoolManagerTest.java

```diff
@@ -212,19 +213,21 @@ public void init() throws Exception {
 
         Map<String, EntitlementCertificate> entCerts = new HashMap<>();
         entCerts.put(pool.getId(), new EntitlementCertificate());
-        when(mockECGenerator.generateEntitlementCertificates(any(Consumer.class), any(Map.class), any(Map
-            .class), any(Map.class), eq(false))).thenReturn(entCerts);
+        when(mockECGenerator.generateEntitlementCertificates(any(Consumer.class), any(Map.class),
+            any(Map.class), any(Map.class), eq(false))).thenReturn(entCerts);
         dummyComplianceStatus = new ComplianceStatus(new Date());
-        when(complianceRules.getStatus(any(Consumer.class), any(Date.class))).thenReturn(
-            dummyComplianceStatus);
-
-        when(consumerCuratorMock.lockAndLoad(any(Consumer.class))).thenAnswer(new Answer<Consumer>() {
-            @Override
-            public Consumer answer(InvocationOnMock invocation) throws Throwable {
-                Object[] args = invocation.getArguments();
-                return (Consumer) args[0];
-            }
-        });
+        when(complianceRules.getStatus(any(Consumer.class), any(Date.class)))
+            .thenReturn(dummyComplianceStatus);
+
+        // Make sure lock methods are passthrough
+        doAnswer(returnsFirstArg()).when(this.mockPoolCurator).lock(any(Pool.class));
+        doAnswer(returnsFirstArg()).when(this.mockProductCurator).lock(any(Product.class));
+        doAnswer(returnsFirstArg()).when(this.entitlementCurator).lock(any(Entitlement.class));
+        // doAnswer(returnsFirstArg()).when(this.certCuratorMock).lock(any(Persisted.class));
+        doAnswer(returnsFirstArg()).when(this.consumerCuratorMock).lock(any(Consumer.class));
+        doAnswer(returnsFirstArg()).when(this.mockOwnerCurator).lock(any(Owner.class));
+        // doAnswer(returnsFirstArg()).when(this.mockOwnerContentCurator).lock(any(OwnerContent.class));
+        // doAnswer(returnsFirstArg()).when(this.mockOwnerProductCurator).lock(any(OwnerProduct.class));
```

Comment:
Commented out code can be removed (unless there is a reason for it, which could be explained in a comment).